packages/components/src/guide/style.scss

```
@@ -93,11 +93,14 @@
 	max-height: 575px;
 
 	@media (max-width: $break-small) {
+		position: absolute;
 		bottom: 5%;
 		left: $grid-unit-20;
 		right: $grid-unit-20;
 		top: 5%;
 		margin: 0 auto;
+		min-width: 280px;
+		max-width: calc(100vw - #{$grid-unit-20} * 2);
```

```suggestion
		margin: auto;
		min-width: 280px;
		max-width: calc(100vw - #{$grid-unit-20} * 2);
		
		 // This can be avoided if we add styles to let the image resize or don't change the min-width
		&.components-modal__content {
			overflow-x: hidden;
		}
```
Using auto margins instead of positioning is simpler and goes along with the style refactor from #31639. Removal of `position: absolute` means the footer rules needs an accompanying change but as mentioned in my other comment I think that is already wanting.